congruity/rdd_adapter.py

```diff
@@ -151,6 +151,12 @@ def collect(self: "RDDAdapter[T]") -> List[T]:
             return [self._unnest_data(row[0]) for row in data]
         return data
 
+    def collectAsMap(self: "RDDAdapter[Tuple[K, V]]") -> Dict[K, V]:
+        return dict(self.collect())
```

Comment:
I'm wondering if this needs an additional assertion if the input RDD is not of tuple form.

Reply:
Addressed please review